internal/scanner/scanner.go

```diff
@@ -131,12 +130,11 @@ func NewScanner(
 // logs them if there's a problem auditing the resource of saving the Report or
 // Result, so it can continue with the next audit, or next Result.
 func (s *Scanner) ScanNamespace(ctx context.Context, nsName, runUID string) error {
-	log.Info().
-		Dict("dict", zerolog.Dict().
-			Str("namespace", nsName).
-			Str("RunUID", runUID).
-			Int("parallel-resources-audits", s.parallelResourcesAudits),
-		).Msg("namespace scan started")
+	s.logger.InfoContext(context.Background(), "namespace scan started",
```

Comment:
this is valid for all the other logger calls, we need to use the context from the method parameter.

```go
s.logger.InfoContext(ctx, ....)
```